audio_clip_encoder.py

```diff
@@ -55,10 +53,8 @@ def _create_embeddings(self, filtered_docs: Iterable):
         """Update the documents with the embeddings generated by AudioCLIP"""
 
         for d in filtered_docs:
-            audio = [d.blob, deepcopy(d.blob)]
-            audio = torch.stack([self.audio_transforms(track.reshape(1, -1)) for track in audio])
+            track = d.blob
+            audio = torch.stack([self.audio_transforms(track.reshape(1, -1))])
```

Comment:
are we sure `audio` cannot work in `batches`?

Reply:
so you want to stack all document blobs together and encode them at once ?

Reply:
not all, but work in batches, like with other executors

Reply:
should be done